Approximate_convex_decomposition/include/CGAL/approx_decomposition.h

```diff
@@ -0,0 +1,134 @@
+#ifndef CGAL_APPROXIMATE_CONVEX_DECOMPOSITION_APPROX_DECOMPOSITION_H
```

Comment:
Add missing GPL license headers to all headers. See https://doc.cgal.org/latest/Manual/devman_code_format.html#secfile_header

Reply:
+ you need to add the license include. I suggest that we integrate your work into the Surface mesh segmentation package because in the end, this is a segmentation method. See how a header of this package looks like to mimic it.